test/oas3/execute/style-explode/query.js

```diff
@@ -699,7 +699,7 @@ describe('OAS 3.0 - buildRequest w/ `style` & `explode` - query parameters', ()
 
         expect(req).toEqual({
           method: 'GET',
-          url: `/users?id=3 id=4 id=5 id=${SAFE_INPUT_RESULT}`,
+          url: `/users?id=3&id=4&id=5&id=${SAFE_INPUT_RESULT}`,
```

Comment:
There is no example when style is spaceDelimited or pipeDelimited and explode is true, but the same behavior as when style is form is probably correct.

https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#style-examples